can_tools/scrapers/official/DE/de_vaccine.py

```diff
@@ -146,6 +145,9 @@ def normalize(self, data: Dict[str, requests.models.Response]) -> pd.DataFrame:
                 age=lambda row: row["age"].str.replace("+", "_plus"),
                 dt=lambda row: row["dt"].dt.date,
             )
-            .replace(["patient_declined_to_disclose", "data_not_reported"], None)
```

Comment:
this was causing some duplicate rows, so I replaced it with the code below and verified that the data was correct